File: 2706-buy-two-chocolates/2706-buy-two-chocolates.py
class Solution(object):
    def buyChoco(self, prices, money):
        """
        :type prices: List[int]
        :type money: int
        :rtype: int
        """
        min1 = float('inf')
        min2 = float('inf')

        for price in prices:
            if price < min1 :
                min2 = min1
                min1 = price
            elif price < min2 :
                min2 = price
        
        remain = money - (min1 + min2)

        if(remain >= 0) : 
            return remain
        return money
        

File: 2706-buy-two-chocolates/2706-buy-two-chocolates.cpp
class Solution {
public:
    int buyChoco(vector<int>& prices, int money) {
        int min1 = INT_MAX ;
        int min2 = INT_MAX ;

        for(int price : prices) {
            if(price < min1) {
                min2 = min1;
                min1 = price;
            }
            else if (price < min2) {
                min2 = price;
            }
        }
        int remain = money - (min1 + min2);

        if(remain >= 0) return remain;
        return money;
    }
};